### PR TITLE
Fix gridrec filter scaling

### LINKED ISSUE #209: Gridrec and FBP results are off from expected
We tested the tomopy recon algorithms with a phantom, and are not getting the expected results for pixel intensity values for gridrec and FBP (all other algorithms agree with what we expect). Gridrec pixel intensity values are off by a factor of approximately 1/2, and the FBP values are off by several orders of magnitude.

### PR DESCRIPTION
**Pull request by dmpelt**
This PR potentially fixes #209 and #132 by fixing the scaling of the gridrec filters. They seemed to be off by a factor of 2 in some cases. 

I basically compared with ASTRA's filter definitions, which includes the factor of 2 (https://github.com/astra-toolbox/astra-toolbox/blob/master/cuda/2d/fft.cu#L311). I tried to match the filter definitions of both TomoPy and ASTRA, but please check whether I made any mistakes.

I am definitely not sure about the Parzen and Butterworth filter definitions, as the Parzen definition in ASTRA seems quite different, and butterworth is not included in astra. Do you have specific mathematical definitions that you used when implementing them?

### CODE REVIEW

**Comment by dmpelt** [comment]
I've activated the tests. I had to recreate the `.npy` data files, so at the moment it doesn't really test anything: the `npy` files were created with the same version. For future versions it will test whether something changed in the gridrec implementation.

**Comment by dgursoy** [comment]
Ok, I will wait till npy  files are replaced then. For consistency, you can use proj.npy for the reproduction of the reconstructions.

**Comment by dmpelt** [comment]
The new npy files should already be included in the PR (I used the `proj.npy` file to create them), so if you think I didn't make any mistakes I guess we can merge this. 

Of course, this PR does change reconstruction results for gridrec (which many people use), so should we include a notification or warning to users as well?

**Comment by dgursoy** [comment]
I am merging this one but could you summarize changes in a list at: 
https://github.com/tomopy/tomopy/tree/master/doc/source/release/notes-1.1.0

It'll probably make our lives easier later on. Thanks!
